whois-update/src/main/java/net/ripe/db/whois/update/handler/validator/common/MemberByRefValidator.java

```diff
@@ -0,0 +1,87 @@
+package net.ripe.db.whois.update.handler.validator.common;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Sets;
+import net.ripe.db.whois.common.Message;
+import net.ripe.db.whois.common.dao.RpslObjectDao;
+import net.ripe.db.whois.common.dao.RpslObjectInfo;
+import net.ripe.db.whois.common.domain.CIString;
+import net.ripe.db.whois.common.rpsl.AttributeType;
+import net.ripe.db.whois.common.rpsl.ObjectType;
+import net.ripe.db.whois.common.rpsl.RpslObject;
+import net.ripe.db.whois.update.domain.Action;
+import net.ripe.db.whois.update.domain.PreparedUpdate;
+import net.ripe.db.whois.update.domain.UpdateContext;
+import net.ripe.db.whois.update.domain.UpdateMessages;
+import net.ripe.db.whois.update.handler.validator.BusinessRuleValidator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.stereotype.Component;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+@Component
+public class MemberByRefValidator implements BusinessRuleValidator {
```

Comment:
Rename to use plural `MembersByRefValidator` to match the attribute `mbrs-by-ref`